Sources/SwiftParser/Types.swift

```diff
@@ -1079,6 +1176,13 @@ extension Parser {
         } else {
           break SPECIFIER_PARSING
         }
+      } else if self.at(.keyword(.nonisolated)) {
+        // If '(' is located on the new line 'nonisolated' cannot be parsed
+        // as a specifier.
+        if self.peek(isAt: .leftParen) && self.peek().isAtStartOfLine {
```

Comment:
```suggestion
        if self.peek(isAt: TokenSpec(.leftParen, allowAtStartOfLine: false)) {
```

Reply:
This is actually not equivalent to what is written, we'd like to stop only if the next token is `(` on the new line, it's not the same condition as above.

Reply:
Sorry, please ignore my comment

Reply:
No worries!